config.go

```diff
@@ -34,6 +35,10 @@ type Job struct {
 	SourceRootURL   string `yaml:"from"`
 	TargetContainer string `yaml:"to"`
 	TargetPrefix    string `yaml:"-"`
+	CertFile	string `yaml:"cert"`
+	KeyFile		string `yaml:"key"`
+	CaFile		string `yaml:"ca"`
```

Comment:
I would like variable names that are a bit more readable, e.g. `ClientCertificatePath`. Especially `CaFile` looks wonky; that could be `ServerCAPath`.